docs/spec_draft.md

```diff
@@ -167,6 +167,7 @@ described below)
    * [or](#stablehloor)
```

Comment:
Also align prose to 80 characters (see other sections in the file). Examples can be longer than 80 characters. We don't have a linter for this yet, but this is what we're informally following in the spec draft.